CHANGELOG.md

```diff
@@ -12,29 +15,39 @@
   - [Changed] no longer need to instantiate a client and manually pass it to the Archive
   - [Changed] rename `run_with` to `drive`
   - [Changed] Archive now accepts just a postgres URL instead of a postgres URL split into its parts. This should
-  make configuring the archive more straightforward. Takes from environment variable `DATABASE_URL` if not passed to the 
+  make configuring the archive more straightforward. Takes from environment variable `DATABASE_URL` if not passed to the
   archive directly
 - [Added] Archive now reads the `CHAIN_DATA_DB` environment variable if the path to the backend chain database is not passed directly.
-- [Changed] `drive` changed from sync to async function
 - [Removed] Archive no longer needs an RPC url to function
+- [Changed] Printing out of Postgres URL in plain text
+- [Added] Maximum amount of blocks loaded at once can be configured with `max_block_load`
+- [Added] Two options to configuration file for State Tracing
```

Comment:
```suggestion
- [Added] Two new options to the configuration relative to state tracing:
```

I think we should explain what state tracing actually is somewhere. It's the big feature of this release, so worth spending some text on it imo. (A link to the wiki might suffice here)

Reply:
Added link to https://github.com/paritytech/substrate-archive/wiki/6.)-State-Tracing-&-Balance-Reconciliation